.github/workflows/ci.yml

```
@@ -202,6 +214,11 @@ jobs:
           start: npm start
           wait-on: http://localhost:3000
           spec: cypress/integration/e2e/**/*
+          record: true
+          tag: node-${{ matrix.node }}
+        env:
+          CYPRESS_RECORD_KEY: "a25e3b24-8ff8-41e6-be40-5e33e33596e2"
```

@ramrodo is this a key that we might store in a GitHub secret?

---

That is a good idea @leecalcote! I don't have permissions on Github to do that yet. Could you help me with that?

---

```suggestion
          CYPRESS_RECORD_KEY: ${{ secrets.CYPRESS_RECORD_KEY }}
```

Perfect. I have added this as an organization-level secret. 👍